lang/rust/avro/src/encode.rs

```diff
@@ -134,6 +134,9 @@ pub(crate) fn encode_internal<S: Borrow<Schema>>(
                     return Err(Error::GetEnumSymbol(s.clone()));
                 }
             }
+            Schema::Uuid => {
```

Comment:
You should be able to add this condition to the above. L 126

```
Schema::String | Schema::Uuid => ....
```

Reply:
Agreed!
@RikHeijdens Please update the PR with @jklamer's suggestion.

Reply:
Updated the PR with @jklamer's suggestion in https://github.com/apache/avro/pull/1778/commits/f0203f0ff4b46bb008c5c89d6856abc45c10cf3b!